app/views/kits/_table.html.erb

```diff
@@ -34,13 +34,6 @@
               </tr>
             <% end %>
           <% else %>
-            <% kit.inventory_items.map do |inventory_item| %>
```

Comment:
@dorner maybe I'll need a walk through of why we need the empty-else or the condition at all

Reply:
I don't think we do!